### ISSUE TITLE
typemap is not being sent correctly when there are multiple servers

### ISSUE THREAD

**Issue by arthur00** [Mondego/spacetime#55]
Spacetime log:

```
[spacetime.store] Application TrafficSimulationArchit disconnected. Removing owned objects.
[spacetime.cache] Type Route is not registered to be obtained by App NodeSimulation.
[spacetime.cache] Type ArchitCar is not registered to be obtained by App NodeSimulation.
[spacetime.store] Object ID f1fd9f59-8819-452c-8631-3c34b0d57b59 missing from data storage.
[spacetime.store] Application TrafficSimulationArchit disconnected. Removing owned objects.
[spacetime.cache] Type ArchitCar is not registered to be obtained by App NodeSimulation.
[spacetime.cache] Type Route is not registered to be obtained by App NodeSimulation.
[spacetime.store] Object ID 0ceb3477-c14b-4882-9043-08c65bcf0502 missing from data storage.
[spacetime.store] Application TrafficSimulationArchit disconnected. Removing owned objects.
[spacetime.cache] Type ArchitCar is not registered to be obtained by App NodeSimulation.
[spacetime.cache] Type Route is not registered to be obtained by App NodeSimulation.
[spacetime.store] Object ID 33f18443-31fe-435f-9ed6-21fcc120bd16 missing from data storage.
[spacetime.store] Application TrafficSimulationArchit disconnected. Removing owned objects.
[spacetime.cache] Type ArchitCar is not registered to be obtained by App NodeSimulation.
[spacetime.cache] Type Route is not registered to be obtained by App NodeSimulation.
```

**Comment by arthur00** [comment]
Addressed in a605e56883676115435a7613e679137f1df05f49 and abc0d562e1acedbe35a75e33c1cfe0dd341c6242. Still needs some testing to check if problem is gone.

**Comment by arthur00** [comment]
No issues so far.. Closing
